rs/rosetta-api/icp/tests/system_tests/test_cases/neuron_management.rs

```diff
@@ -288,3 +289,103 @@ fn test_start_and_stop_neuron_dissolve() {
         ));
     });
 }
+
+#[test]
+fn test_change_auto_stake_maturity() {
+    let rt = Runtime::new().unwrap();
+    rt.block_on(async {
+        let env = RosettaTestingEnvironment::builder()
+            .with_initial_balances(
+                vec![(
+                    AccountIdentifier::from(TEST_IDENTITY.sender().unwrap()),
+                    // A hundred million ICP should be enough
+                    icp_ledger::Tokens::from_tokens(100_000_000).unwrap(),
+                )]
+                .into_iter()
+                .collect(),
+            )
+            .with_governance_canister()
+            .build()
+            .await;
+
+        // Stake the minimum amount 100 million e8s
+        let staked_amount = 100_000_000u64;
+        let neuron_index = 0;
+        let from_subaccount = [0; 32];
+
+        env.rosetta_client
+            .create_neuron(
+                env.network_identifier.clone(),
+                &(*TEST_IDENTITY).clone(),
+                RosettaCreateNeuronArgs::builder(staked_amount.into())
+                    .with_from_subaccount(from_subaccount)
+                    .with_neuron_index(neuron_index)
+                    .build(),
+            )
+            .await
+            .unwrap();
+        env.pocket_ic.tick().await;
```

Comment:
I tried removing this, and the other explicit calls to `env.pocket_ic.tick().await` in this test, and the test still passes with `--runs_per_test=100`. Did you observe some failures, and if so, would it be possible to e.g., keep calling `tick()` in a loop until some expected state is reached? Since pocket IC is running in live mode here, having a single call to `tick()` making a difference seems potentially flaky to me.

Reply:
I added the ticks as a response to a recent [comment](https://dfinity.slack.com/archives/C0175CHJ0P6/p1729600776045439?thread_ts=1729598619.187999&cid=C0175CHJ0P6) from one of the NNS team, that I might have to call tick a few times for the governance canister to propagate the state changes.

Reply:
That thread seems to be related to another test. Additionally, when pocket IC is in live mode, `advance_time` and `tick` are [already called multiple times per second](https://github.com/dfinity/ic/blob/HEAD/packages/pocket-ic/HOWTO.md#live-mode). If some specific amount of time/number of rounds need(s) to pass for governance to make progress, would it make sense to stop the live mode and explicitly advance the expected number of seconds/rounds?

Reply:
Makes sense, I removed the ticks.